WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/FilterableCategoriesViewController.swift

```diff
@@ -108,11 +122,24 @@ class FilterableCategoriesViewController: CollapsableHeaderViewController {
     }
 }
 
-extension FilterableCategoriesViewController: UITableViewDataSource {
+// MARK: - UITableViewDelegate
+
+extension FilterableCategoriesViewController: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let size = isLoading ? ghostThumbnailSize : visibleSections[indexPath.row].thumbnailSize
+        return size.height + CategorySectionTableViewCell.cellVerticalPadding
+    }
 
     func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
```

Comment:
There's probably no big win with this delegate method since we're not dealing with hundreds of rows, but I don't think it hurts to include it.